.gitignore

```diff
@@ -5,4 +5,6 @@ roi-hunter-easy-for-woocommerce.zip
 .vscode
 todo.txt
 roi-hunter-easy_staging.zip
-roi-hunter-easy-for-woocommerce_staging.zip
\ No newline at end of file
+roi-hunter-easy-for-woocommerce_staging.zip
+assets/**/*.min.css
```

Comment:
Nevím nakolik je praktické tyhle soubory ignorovat - když si někdo stáhne "nejaktuálnější" verzi z githubu, tak mu nemusí nutně fungovat, protože si nestáhl balíčky a nezkompiloval CSS, já bych to asi pro jistotu verzovala

Reply:
Myslím si, že repozitář by neměl obsahovat minifikované assety, pokud někdo chce verzi pluginu připravenou k instalaci může si stáhnout balíček z https://github.com/roihunter/roi-hunter-easy-woo-commerce-plugin/releases

@janliska Můžeš prosím přidat do Readme informaci jak si ze zdrojových kódů zbuildit balíček a odkaz na releases?